frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.scss

```diff
@@ -0,0 +1,9 @@
+.odc-toplogy-filter-bar {
+  background-color: var(--pf-chart-global--Fill--Color--white);
```

Comment:
The PF topology layout doesn't give us a white background!?!

cc @jeff-phillips-18

Reply:
No, PF topology doesn't give a white background.

Reply:
Perhaps the question we should be asking is why is it not white behind? Probably due to NamespacePage.... Maybe we should look fixing whatever is causing us to need to add a white background.
That said, maybe we should raise an issue against PF as I think it should be made white. I cannot see why else it makes sense to have it transparent.